File: app/src/main/java/com/thewizrd/simpleweather/updates/UpdateInfo.java
package com.thewizrd.simpleweather.updates;

import com.google.gson.annotations.SerializedName;

public class UpdateInfo {
    @SerializedName("version")
    private int versionCode;
    @SerializedName("updatePriority")
    private int updatePriority;

    public UpdateInfo() {
    }

    public int getVersionCode() {
        return versionCode;
    }

    public void setVersionCode(int versionCode) {
        this.versionCode = versionCode;
    }

    public int getUpdatePriority() {
        return updatePriority;
    }

    public void setUpdatePriority(int updatePriority) {
        this.updatePriority = updatePriority;
    }
}

File: app/src/main/java/com/thewizrd/simpleweather/updates/UpdateInfo.kt
package com.thewizrd.simpleweather.updates

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
class UpdateInfo {
    @Json(name = "version")
    var versionCode = 0

    @Json(name = "updatePriority")
    var updatePriority = 0
}